src/foam/nanos/auth/Address.js

```diff
@@ -22,7 +22,15 @@ foam.CLASS({
   ],
 
   messages: [
-    { name: 'INVALID_POSTAL_CODE_ERR_MSG', message: 'Invalid postal code' }
+    { name: 'CITY_REQUIRED', message: 'City required' },
+    { name: 'COUNTRY_REQUIRED', message: 'Country required' },
+    { name: 'INVALID_ADDRESS_1', message: 'Invalid value for address 1' },
+    { name: 'INVALID_POSTAL_CODE_ERR_MSG', message: 'Valid Postal Code or ZIP Code required' },
+    { name: 'PROVINCE_REQUIRED', message: 'Province required' },
```

Comment:
I don't think we should have PROVINCE_REQUIRED and STATE_REQUIRED. Just use a messageMap for REGION_REQUIRED.